chia/full_node/weight_proof.py

```diff
@@ -20,6 +20,7 @@
     calculate_sp_iters,
     is_overflow_block,
 )
+from chia.util.chunks import chunks
```

Comment:
Belongs down a bit, GitHub won't let me make a suggestion for this one.

Reply:
does `isort` not fix this?
If not, what rule are we following?

Reply:
`isort` is configured to ignore presently-not-compliant files.  mypy has a similar setup.  We just have giant lists of exceptions basically so new files will be clean and we can go through existing files one at a time, clean them, then remove them from the exceptions list.